keyboards/dz60/keymaps/twschum_b_4_10/keymap.c

```diff
@@ -0,0 +1,137 @@
+#include QMK_KEYBOARD_H
+#include "twschum.h"
+
+/* DZ60 layout using following options (from layouts diagram on KBDfans):
+ *  - plate B (2.25u lshift)
+ *  - opt 4 (1.75, 1, 1 on rshift)
+ *  - opt 10 (2.75, 1.25, 2.25 on space, 5x1u bottom right keys)
+ * http://www.keyboard-layout-editor.com/#/gists/225f0f4dcf6671405f744fabe314627c
+ */
+
+/* TODO: provide a standalone B_4_10 layout
+ */
+
+#undef KEYMAP
+#define KEYMAP( \
```

Comment:
If you could, replace all of the `KEYMAP`s with `LAYOUT`.  We've switched terminologies, since `LAYOUT` is more accurate based on what is happening.

Reply:
Will do!